torch/_torch_docs.py

```diff
@@ -7780,12 +7780,17 @@ def merge_dicts(*dicts):
            r"""
 sgn(input, *, out=None) -> Tensor
 
-For complex tensors, this function returns a new tensor whose elemants have the same angle as that of the
-elements of :attr:`input` and absolute value 1. For a non-complex tensor, this function
-returns the signs of the elements of :attr:`input` (see :func:`torch.sign`).
+Returns a new tensor with the signs (definition below) of the elements of :attr:`input`.
 
-:math:`\text{out}_{i} = 0`, if :math:`|{\text{{input}}_i}| == 0`
-:math:`\text{out}_{i} = \frac{{\text{{input}}_i}}{|{\text{{input}}_i}|}`, otherwise
+.. math::
+    \text{out}_{i} = \begin{cases}
+                    0 & |\text{{input}}_i| == 0 \\
+                    \frac{{\text{{input}}_i}}{|{\text{{input}}_i}|} & \text{otherwise}
+                    \end{cases}
+
+.. note::
+    For complex tensors, this function returns a new tensor whose elements
+    have the same angle as that of the elements of :attr:`input` and absolute value 1.
```

Comment:
"same angle as that of the elements" -> "same angles as the corresponding elements"

"and absolute value 1." -> "and absolute values (i.e. magnitudes) of one."